### ISSUE TITLE
Manipulate AnnData title changes not working

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/galaxy#11516]
@mtekman Tool not allowing title change with any version newer than: (Galaxy Version 0.6.22.post1+galaxy3)
History to show the issue: https://humancellatlas.usegalaxy.eu/u/wendi.bacon/h/manipulate-anndata-is-not-functioning

**Comment by mtekman** [comment]
Yep looks like the sniffer is trying to retrieve too much information.

I'll make a fix that reduces exactly how much information is served/editable to the user

**Comment by mtekman** [comment]
I see that it *does* eventually work, it just requires waiting over 10 minutes for the page to retrieve all the data. Could be more of a general galaxy problem than just AnnData now I think about it.

Even so, what is currently served to the user is too much so I'll simplify this later

**Comment by nomadscientist** [comment]
I can add to this that once it's been through the Manipulate AnnData,
further steps (Like AnnData operations) also fail to allow dataset renaming.
